pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
 dependencies:
   xml: ">=4.0.0 <5.0.0"
```

Comment:
Should we increase the xml dependency version to force pull of newer version? I think this is the actual issue behind https://github.com/Viktoriaschule/dart-nextcloud/issues/11#issuecomment-704626644 and it happens when you have an older version in your cache.

Reply:
Sounds alright